src/helpers/Log.hpp

```diff
@@ -18,7 +18,6 @@ enum eLogLevel {
         Debug::log(CRIT, "\n==========================================================================================\nASSERTION FAILED! \n\n{}\n\nat: line {} in {}",            \
                    std::format(reason, ##__VA_ARGS__), __LINE__,                                                                                                                   \
                    ([]() constexpr -> std::string { return std::string(__FILE__).substr(std::string(__FILE__).find_last_of('/') + 1); })().c_str());                               \
-        printf("Assertion failed! See the log in /tmp/hypr/hyprland.log for more info.");                                                                                          \
```

Comment:
??

Reply:
Since Debug::log will log to stdout and the info about `/tmp/hypr/hyprland.log` is not accurate this printf seemed useless.
Removed it because i added an RASSERT for PUTC.